TestResultSummaryService/parsers/Test.js

```diff
@@ -58,25 +58,36 @@ class Test extends Parser {
 
                 testStr = "";
                 testName = m[1];
+                console.log("testname!    " + testName)
                 testResultRegex = new RegExp(testName + "_(.*)\r?");
             }
             if ((m = line.match(regexStartTime)) !== null) {
-                startTime = m[1];
+                startTestName = m[1];
+                startTime = m[2];
+                console.log("startTestName   "+ startTestName);
+                console.log("starttime    " + startTime); 
             }
             if (testResultRegex && ((m = testResultRegex.exec(line)) !== null)) {
                 testResult = m[1];
             }
             if (testName) {
                 testStr += line + "\n";
                 if ((m = line.match(regexFinishTime)) !== null) {
+                    finishTestName = m[1];
+                    finishTime = m[2];
+                    console.log("finishTestName   "+ finishTestName);
+                    console.log("finishTime    " + finishTime); 
+                    const durationTime = (startTestName == finishTestName && testName == startTestName && 
+                        testName == finishTestName && startTime > 0 && finishTime > 0 && 
+                        finishTime - startTime > 0)? finishTime - startTime : null;

```

Comment:
Since we checked the values above, we just need to check if finishTime - startTime > 0